<!-- -*- mode: XML -*- -->
<!-- start vvelodyne light node -->

<launch>
  <arg name="device_ip" default="" />
  <arg name="frame_id" default="velodyne" />
  <arg name="model" default="VLP16" />
  <arg name="pcap" default="" />
  <arg name="port" default="2368" />
  <arg name="rpm" default="600.0" />
  <arg name="gps_time" default="false" />
  <arg name="beforeZeroAng" value="-1"/>
  <arg name="afterZeroAng" value="-1"/>
  <arg name="timestamp_first_packet" default="false" />

<!-- TODO: Da modificare -->

  <!-- Starting lidar node -->
  <node pkg="velodyne_light" type="velodyne_light_node" name="velodyne_light" output="screen">
    <param name="device_ip" value="$(arg device_ip)" />
    <param name="frame_id" value="$(arg frame_id)"/>
    <param name="model" value="$(arg model)"/>
    <param name="pcap" value="$(arg pcap)"/>
    <param name="port" value="$(arg port)" />
    <param name="rpm" value="$(arg rpm)"/>
    <param name="gps_time" value="$(arg gps_time)"/>
    <param name="beforeZeroAng" value="$(arg beforeZeroAng)"/>
    <param name="afterZeroAng" value="$(arg afterZeroAng)"/>
    <param name="timestamp_first_packet" value="$(arg timestamp_first_packet)"/>
  </node>

</launch>
